src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,137 @@
+package duke;
+
+import duke.commands.ByeCommand;
+import duke.commands.Command;
+import duke.commands.DateCommand;
+import duke.commands.DeadlineCommand;
+import duke.commands.DeleteCommand;
+import duke.commands.EventCommand;
+import duke.commands.FindCommand;
+import duke.commands.ListCommand;
+import duke.commands.MarkCommand;
+import duke.commands.TodoCommand;
+import duke.commands.UnmarkCommand;
+
+
+/**
+ * The Parser class deals with making sense of the user command.
+ */
+public class Parser {
+
+    /**
+     * Make sense of the user's input by parsing.
+     * @param userInput The command user input in Duke command line.
+     * @return A Command object representing user command.
+     * @throws DukeException when user command is invalid.
```

Comment:
There should be an empty line between description and parameter section. I noticed the same issue in several other places too.